### DISCUSSION TITLE
Rustdesk 1.2.4 STEALTH MODE NOT WORKING? (hide cm)

### DISCUSSION THREAD

**Discussion by MoHara23** [rustdesk/rustdesk#6117] [Q&A]
Dear

Each time i connect, a popup on remote computer shows that connection is established, but I remember in previous version a setting to change this behaviour, I think under security, but it isnt there anymore.

Some one?

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/discussions/6008

**Comment by RichardKimJH** [comment]
"Hide the Connection Manager window"
I need this too.
How about a small notification window in the tray location, like "tray notifications"?
